vtu11/impl/vtu11_impl.hpp

```diff
@@ -146,6 +150,43 @@ void write( const std::string& filename,
   output.close( );
 }
 
+//ParallelWrite generates a pvtu file and accordingly the vtu pieces in a subfolder
+//Each piece consists of a set of points and cells
+template<typename MeshGenerator, typename Writer = AsciiWriter>
+void parallelWrite( const std::string& path,
+                    std::string baseName,
+                    MeshGenerator& mesh,
```

Comment:
This is the mesh no?
=> why is it not `const`?
=> why is it called MeshGenerator and not simply e.g. `Mesh`?
Maybe @phmkopp can comment, not sure if this was there initially

Reply:
Because its a template type that is required to provide functions that return data. It could be a proxy around an actual mesh that simply converts the data structures when needed.

The reason it's not const is that this process of providing data on demand (which I originally imagined a bit like a python generator) might require to change state. Like for example a constant iterator can still be incremented. 

We never followed up on this idea of not copying the entire data structure, but that's where it comes from.

Reply:
makes sense, I forgot abt that